setup.cfg

```diff
@@ -1,3 +1,43 @@
+[metadata]
```

Comment:
While this doesn't look that bad (actually quite nice), I would prefer to keep the projects within the python-hyper org as similar as possible, especially for all HTTP/2 projects: h2, hpack, hyperframe, and now priority.

Do you have strong arguments or feels for stripping `setup.py` and moving it to `setup.cfg`? And what about `pyproject.toml`, I'd rather not open this topic and stay with the "traditional" way for now.

Reply:
setuptools strongly recommends using the declarative metadata for configuration. I'll obviously move everything over to PEP 621 metadata once ready

Reply:
It's also easier to move to PEP 621 metadata once everything is in a declarative format, as you can use setuptools' `parse_configuration()` and `toml.dumps`